workflows/subwf-atac.wdl

```diff
@@ -1,8 +1,7 @@
 version 1.0
 
 # Import the tasks called by the pipeline
-import "../tasks/share_task_correct_fastq.wdl" as task_correct_fastq
-import "../tasks/task_trim_fastqs_atac.wdl" as task_trim
+import "../tasks/task_chromap_read_format.wdl" as task_chromap_read_format
```

Comment:
oh you do have chromap here. so call it get_chromap_read_format and change it in the WDL as well